src/local/service.rs

```diff
@@ -75,9 +75,9 @@ impl LocalService {
         let mut images = Vec::new();
         if let Some(image_url) = metadata.get("image").and_then(|img| img.as_str()) {
             images.push(PlaytronImage {
-                image_type: "capsule".to_string(),
+                image_type: "landscape".to_string(),
```

Comment:
Do we need to add a fallback to capsule if landscape doesn't work? Or is that handled elsewhere (by Grid?)?

Reply:
It's handled by both Grid and Playserve dependending on how we manage images. 
Making the plugin deal with this would involve making lots of HTTP requests and could make the library import much longer.

Reply:
Ideally, we shouldn't use the capsule format since it's meant for the Steam store and not the Steam client. Capsule images often are modified from the official artwork with stuff like "voted game of the year by IGN" or "now with new DLC". Once we switch to hero + logo, the usage of capsule should be much smaller.